prettier.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Pattern in our ESLint configs seems to be to make tool-specific configs additive (or... remove-ative?) rather than have them extend the base config directly (except for React, which we might want to change eventually)

Reply:
I considered that first, but I don't we have enough tooling in the stylelint repo to really benefit from that pattern.